Updated TrustedHosts commands

SteveBurkettNZ opened the pull request:
Added required apostrophe's around string for computer names, added Concatenate parameter to preserve any pre-existing values in TrustedHosts list rather then overwrite them.

azurecla commented:
Hi __@ahgyger__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ahgyger). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ahgyger commented:
#sign-off

PRmerger commented:
@ahgyger : Thanks for your contribution to the Azure documentation! The author, @ahgyger, has been notified to review your proposed change.

PRmerger commented:
Microsoft authors of articles need to make updates through the private repository so the updates can be staged and validated by the current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged.

It�s easy to access the private repo version of your articles in GitHub. Just add �pr to the URL, as follows: 

Public repo URL: https://github.com/Azure/azure-content/blob/master/articles/azure-portal-overview.md

Private repo URL: https://github.com/Azure/azure-content-pr/blob/master/articles/azure-portal-overview.md

We've closed this pull request. Please resubmit the updates through the private repo. If you need perms to the private repo, see the contributor guide instructions.

SteveBurkettNZ committed:
Updated TrustHosts commands

Added required apostrophe's around string for computer names, added Concatenate parameter to preserve any pre-existing values in TrustedHosts list rather then overwrite them.
